pkg/limits/mem/mem.go

```diff
@@ -1,3 +1,4 @@
+//go:build !linux
```

Comment:
why was this needed?

Reply:
not it may break the other `+build !linux` not sure

Reply:
This is done automatically by go 1.17, and the same being on Prometheus side.

Reply:
From [go 1.17 release notes](https://golang.org/doc/go1.17):

> The go command now understands //go:build lines and prefers them over // +build lines.